mobile/lib/providers/authentication.provider.dart

```diff
@@ -170,8 +170,10 @@ class AuthenticationNotifier extends StateNotifier<AuthenticationState> {
     UserPreferencesResponseDto? userPreferences;
     try {
       final responses = await Future.wait([
-        _apiService.usersApi.getMyUser(),
-        _apiService.usersApi.getMyPreferences(),
+        _apiService.usersApi.getMyUser().timeout(const Duration(seconds: 3)),
+        _apiService.usersApi
+            .getMyPreferences()
+            .timeout(const Duration(seconds: 3)),
```

Comment:
Lets make this longer to avoid potentially making it impossible to connect with a slow connection